commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -22,7 +22,7 @@ cdef extern from "trk2dictionary_c.cpp":
     ) nogil
 
 
-cpdef run( filename_tractogram, path_out, TCK_ref_image = None, filename_peaks = None, filename_mask = None, do_intersect = True,
+cpdef run( filename_tractogram = None, filename_trk = None, path_out = None, TCK_ref_image = None, filename_peaks = None, filename_mask = None, do_intersect = True,
```

Comment:
@MarioOcampo , shouldn't this be `filename_tractogram = None, path_out = None, filename_trk = None` instead? In the way it is implemented, is the positional passing of parameters backward compatible?

Reply:
Yes. it should be at the beginning  `filename_tractogram = None, path_out = None` and as last parameter `filename_trk = None`.